draft-ietf-scitt-architecture.md

```diff
@@ -363,36 +366,41 @@ This section describes at a high level, the three main roles and associated proc
 
-Issuers SHOULD use consistent identifiers for all their Statements about Artifacts, to simplify authorization by Verifiers and auditing.
-If an Issuer uses multiple identifiers, they MUST ensure that statements signed under each identifier are consistent.
+Issuers MAY rotate verification keys at any time and SHOULD rotate verification keys at an appropriate and consistent cryptoperiod (see {{KEY-MANAGEMENT}}).
+Issuers MAY migrate to new signing and verification algorithms, but the Transparency Service remains responsible for admitting Signed Statements that complies with its active Registration Policies.
+The version of the key used to sign the Signed Statement MAY be included via the `kid` COSE header parameter in the protected header. Key discovery protocols are out-of-scope of this document.
 
-Issuers MAY rotate verification keys at any time, or at a consistent cryptoperiod.
-Issuers MAY migrate to new signing and verification algorithms, but the Transparency Service remains responsible for admitting signed statements that match its policies.
+An Issuer identifier is required to be included in the COSE envelope. The protected header of a Signed Statement MUST include the `CWT Claims` header parameter as specified in {{Section 2 of CWT_CLAIMS_COSE}}. The CBOR map that constitutes the corresponding `CWT Claims` value MUST include the `Issuer Claim` (Claim label 1, see {{IANA.cwt}}) where the value MUST represent the Issuer identifier.
 
-The Issuer's identifier is required and appears in the `1 iss` claim of the `15 CWT_Claims` protected header of the Signed Statements' Envelope.
-The version of the key used to sign the Signed Statement is written in the `4 kid` protected header.
+An Subject identifier is also required to be included in the COSE envelope. The CBOR map that constitutes the corresponding `CWT Claims` value MUST include the `Subject Clain` (Claim label 2, see {{IANA.cwt}}) where the value MUST represent the Subject identifier.
 
-Key discovery protocols are out of scope for this document.
+Figure {{fig-signed-statement}} illustrated a normative CDDL definition for the composition of the protected header in COSE envelope of SCITT Signed Statements.
 
 ~~~ cddl
 CWT_Claims = {
-  1 => tstr; iss, the issuer making statements,
-  2 => tstr; sub, the subject of the statements,
-  * tstr => any
+    1 => tstr            ; iss, the issuer making statements
+    2 => tstr            ; sub, the subject of the statements
+  * int => any
 }
 
 Protected_Header = {
-  1   => int             ; algorithm identifier,
-  4   => bstr            ; Key ID (kid),
-  15  => CWT_Claims      ; CBOR Web Token Claims,
-  3   => tstr            ; payload type
+    1 => int             ; algorithm identifier
+    3 => tstr            ; payload type
+  ? 4 => bstr            ; Key ID (kid)
+   15 => CWT_Claims      ; CBOR Web Token Claims
+   34 => COSE_CertHash   ; x5t, hash of an X.509 certificate
```

Comment:
are additional properties allowed here? I think they should be.

Reply:
they were not allowed previously. While that does not make a lot of sense, I kept it that way for now.

Reply:
17h later, it still makes no sense, changed it via suggestion feature.